### DISCUSSION TITLE
【求助】之前去检查发现我的体质特殊想去空洞看看，但是不会做萝卜，有没有大佬给个教程，或者愿意合作的

### DISCUSSION THREAD

**Discussion by HonmaMeiko02** [share121/inter-knot#1069] [其他]
![kondon](https://github.com/user-attachments/assets/fcc7dc39-561c-449e-b1c6-6670689f6244)

**Comment by fairy-B** [comment]
自己不进入空洞，同时实时更新上传修正的萝卜数据（利用H.D.D.），在无特殊情况的时候，可以制定多条行进路线

**Comment by Lanxey** [comment]
【方法一】

1. 拿出手机
2. 下载地图导航App
3. 搜索【菜市场】
4. 抵达菜市场后，找到蔬菜类分区
5. 寻找萝卜购买

【方法二】

1. 拿出手机
2. 下载买菜软件
3. 搜索：萝卜
4. 下单
